images/empty-content.svg

```diff
@@ -0,0 +1 @@
6-1.6-3.6-3.3-5.9-4.8-2.3-1.5-5-3-7.7-4.1-2.8-1.1-5.6-2-8.4-2.6-.7-.1-1.4-.3-2-.4-.5-.1-1-.3-1.5-.5-.9-.5-1.6-1.3-1.9-2.2-.2-.4-.3-.9-.3-1.2v-.7-.7c0-.9 0-1.7-.1-2.3 0-1.3-.1-2-.1-2V71c1-.5 1.9-1.1 2.6-1.8 1.3-1.3 2.2-2.9 2.7-4.5s.8-3.1 1-4.4c.2-1.4.2-2.7.3-3.8v-2.9-2.5c0-.3-.2-.5-.5-.5s-.5.2-.5.5c0 0 0 .9-.1 2.5 0 .8-.1 1.8-.1 2.9-.1 1.1-.2 2.3-.4 3.7-.2 1.3-.5 2.7-1 4.2-.5 1.4-1.3 2.8-2.4 3.9-1.1 1.1-2.6 1.9-4.3 2.3-.4.1-.8.2-1.2.2-.4.1-.8.2-1.3.2l-1.4.1H141.5c-1.8 0-3.5-.3-5.2-.7-1.7-.4-3.1-1.3-4.3-2.3-1.2-1.1-2.1-2.4-2.7-3.8-.4-.7-.5-1.4-.8-2.1-.2-.7-.3-1.4-.5-2-.1-.7-.2-1.3-.3-1.9-.1-.6-.1-1.2-.2-1.7-.1-1.1-.2-2-.2-2.8 0-1.6-.1-2.5-.1-2.5 0-.3-.2-.5-.5-.5s-.5.2-.5.5v2.5c0 .8 0 1.8.1 2.9 0 .6.1 1.2.2 1.8.1.6.2 1.3.3 2 .1.7.3 1.4.5 2.1.3.7.4 1.5.8 2.2.6 1.5 1.6 3 2.9 4.3 1.4 1.2 3.1 2.2 4.9 2.6 1.8.5 3.7.8 5.6.8H142.9l1.4-.1c.4 0 .9-.1 1.4-.2.5-.1 1-.1 1.4-.3l.9-.3v1.5c0 .6 0 1.4-.1 2.3v1.5c0 .6.2 1.2.4 1.8.4 1.2 1.3 2.3 2.6 3.1.6.4 1.3.7 2.1.8.6.1 1.3.3 1.9.4 2.6.6 5.4 1.4 8 2.5 2.6 1.1 5.2 2.4 7.4 3.9s4.2 3 5.8 4.5c1.6 1.5 2.8 2.8 3.6 3.7.8 1 1.3 1.5 1.3 1.5.2.3.6.3.9.1.2-.2.3-.6.1-.9z"/><path fill="#CAC6C8" d="M154.6 52c-.6-.7-1.6-1.2-2.6-1.2l-3.5-.1c-1.4-.1-2.7.6-3.5 1.6h-.2l-5.4-.2c-.1-.2-.2-.3-.4-.5-.8-.8-1.8-1.3-3-1.3l-3.5-.1c-1 0-2 .4-2.6 1.1-.6.7-1 1.5-.9 2.5l.1 1.1c.2 2.5 2.3 4.4 4.9 4.4h1c2.8 0 5-2.1 5.1-4.8 0-.4-.1-.8-.2-1.2l4.5.1c-.1.4-.2.8-.2 1.2-.1 2.6 2 4.9 4.7 5.1l.9.1h.4c2.4 0 4.5-1.7 4.9-4.1l.2-1.1c.2-.9-.1-1.8-.7-2.6zm-19.5 6.1h-.9-.1c-2 0-3.7-1.5-3.8-3.3l-.1-1.1c0-.6.2-1.2.6-1.6.4-.5 1.1-.7 1.7-.7h.1l3.5.1c.8 0 1.6.4 2.2 1 .5.6.8 1.3.8 2-.1 1.9-1.8 3.5-4 3.6zm19.1-3.7l-.2 1.1c-.3 1.9-2.1 3.2-4.1 3.1l-.9-.1c-2.1-.2-3.8-1.9-3.7-3.9.1-1.6 1.5-2.8 3.2-2.8l3.5.1c.7 0 1.3.3 1.7.8.4.5.6 1.1.5 1.7z"/><g fill="#CAC6C8"><path d="M15.5 65.5c.1 0 .1-.1.1-.2-.2-.4-.3-.7-.4-1.1-.1-.4-.1-.7 0-1.1.1-.7.5-1.4 1-1.8l-.3-.4-.3-.4c-.4.3-.7.7-.9 1.1-.2.4-.3.9-.4 1.4 0 .5.1.9.2 1.4l.3.6c.1.2.3.4.4.5h.3zM16.2 61.3c.5-.5 1.3-.7 2-.7.4 0 .7.1 1.1.2.4.1.7.3 1 .6.1 0 .1.1.2 0 .1 0 .1-.1.1-.2-.2-.4-.6-.7-.9-1-.4-.3-.8-.4-1.3-.5-.5-.1-1 0-1.4.1-.5.1-.9.4-1.3.7l.3.5.2.3zM26 61.4c.3-.3.6-.5 1-.6.3-.1.7-.2 1.1-.2.7 0 1.4.2 2 .7l.3-.4.3-.4c-.4-.3-.8-.5-1.3-.7-.5-.1-1-.2-1.4-.1-.5.1-.9.2-1.3.5s-.7.6-.9 1v.2c.1.1.2.1.2 0z"/><path d="M31.6 61.6c-.2-.4-.5-.8-.9-1.1l-.4.5-.2.3c.6.4.9 1.1 1 1.8.1.4.1.7 0 1.1-.1.4-.2.7-.4 1.1v.2c.1.1.2.1.2 0 .2-.1.3-.3.4-.5l.3-.6c.2-.4.3-.9.2-1.4.2-.5 0-.9-.2-1.4zM25 67.3c-.4.5-.8 1.1-1.1 1.7 0-1.2-.1-2.4-.3-3.6 0-.1-.1-.1-.1-.1-.1 0-.2 0-.2.1-.3 1.8-.3 3.6-.3 5.5 0 .2.1.4.3.4.2.1.5 0 .7-.2.6-1.2 1.1-2.4 1.5-3.6 0-.1 0-.1-.1-.2-.3-.1-.4-.1-.4 0z"/><path d="M23.4 61.7c-4.4 0-8 3.6-8 8 0 2.1.8 4 2.2 5.5-.1.4-.2.7-.3 1.1-.3 1.2-.5 2.5-.5 2.5 0 .1 0 .1.1.2.1 0 .2 0 .2-.1 0 0 .6-1.1 1-2.3.1-.2.2-.4.3-.7 1.4 1.1 3.1 1.7 5 1.7s3.6-.7 5-1.7c.1.2.2.4.3.7.5 1.2 1 2.3 1 2.3 0 .1.1.1.2.1s.2-.1.1-.2c0 0-.2-1.3-.5-2.5-.1-.4-.2-.8-.3-1.1 1.4-1.4 2.2-3.4 2.2-5.5-.1-4.4-3.6-8-8-8zm0 15c-3.9 0-7-3.1-7-7s3.1-7 7-7 7 3.1 7 7-3.2 7-7 7z"/></g></svg>
```

Comment:
Is this meant to be [the person in a wheelchair illustration](https://wpcalypso.wordpress.com/calypso/images/extensions/woocommerce/woocommerce-relaxed.svg)? I see the wheel was removed but the handle wasn't, so the chair itself looks a little odd @jameskoster @LevinMedia (the non-wheelchair version [has a sharper angle chair back](https://wpcalypso.wordpress.com/calypso/images/illustrations/checkEmailsDesktop.svg), too)